server.js

```diff
@@ -23,6 +40,42 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec();
+  res.json(thoughts);
+});
+
+app.post('/thoughts', async (req, res) => {
+  const {message, hearts} = req.body;
```

Comment:
You use spaces after the {} and the naming further down (ex. { thoughtID }), try to be consistent all the way through ☺️